### PR TITLE
Add Capture Device Information Example

### PR DESCRIPTION
**Pull request by brendanorourke**
Includes an example of using a JavaScript Action Item to capture device information as a part of a Checkbox survey.

### CODE REVIEW

File: Surveys/capture-device-information/capture-device-information.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+        if (currUserAgent) {
+
+            if (prevUserAgent !== currUserAgent) {
+
+                // Set the value and hide the field from the user.
+                userAgentField.value = currUserAgent;
+                userAgentField.readOnly = true;
+                userAgentField.hidden = true;
+                
+                // Checkbox Frontend uses AngularJS, we need to force a DOM re-render.
```
Comment (PavelKrysenko) [inline Surveys/capture-device-information/capture-device-information.js:20]: We are using Angular, not AngularJS. It is a different framework.

Comment (impurator) [reply Surveys/capture-device-information/capture-device-information.js:20]: @brendanorourke Should be good to merge after you address this minor comment!

Comment (impurator) [reply Surveys/capture-device-information/capture-device-information.js:20]: @brendanorourke I can make the edit if you'd like, just noticed this has been sitting open for a long time 😄

File: Surveys/capture-device-information/capture-device-information.js [hunk 2]
```diff
@@ -17,7 +17,7 @@
                 userAgentField.readOnly = true;
                 userAgentField.hidden = true;
                 
-                // Checkbox Frontend uses AngularJS, we need to force a DOM re-render.
+                // Checkbox Frontend uses Angular, we need to force a DOM re-render.
```
Comment (brendanorourke) [inline Surveys/capture-device-information/capture-device-information.js:20]: @impurator @PavelKrysenko Thank you for the previous comment-- CLEARLY I need to update my GitHub notifications to go to my @checkbox.com email :)

I've updated the comment to reflect the correct framework.

**Review by impurator — APPROVED** [review]

**Pull request merged**